.azurePipeline/unittest_wheel_steps.yml

```diff
@@ -10,18 +11,19 @@ steps:
   displayName: 'Init Python'
 
 - script: |
-      pip install -U codecov -r requirements.txt
+      python -m pip install --upgrade pip
+      pip install -U pytest-azurepipelines codecov -r requirements.txt
   displayName: 'Install testing requirements'
 
 - task: DownloadPipelineArtifact@2
   inputs:
     artifactName: ${{ parameters.artifactName }}
-    patterns: '**/*.whl'
+    patterns: ${{ parameters.artifactPattern }}
     path: $(Pipeline.Workspace)
 
 - script: |
-      pip install $(Pipeline.Workspace)/anonlink*.whl
-  displayName: 'Install anonlink'
+      pip install anonlink --no-index -f $(Pipeline.Workspace)
```

Comment:
By default, the template will pull all the wheel archives. I'm assuming pip will select the good one here.